src/problem.cpp

```diff
@@ -53,6 +54,11 @@ void LogCmdFindConvolution(const miopen::TensorDescriptor& x,
                            std::optional<uint64_t> solver_id);
 } // namespace miopen::debug
 
+// Todo: This should be updated when a separate driver command is implemented
```

Comment:
```suggestion
/// \todo: This should be updated when a separate driver command is implemented
/// for 2.0 Activation API.
```

Reply:
Why? Both main C++ IDEs (MSVS and CLion) and, probably, VSCode detect any todo word in comments.

Reply:
I don't think doxygen produces something from todos on random lines and I'd rather not have it produce a page for a random internal function if it would.

Reply:
@DrizztDoUrden just for unification

Reply:
[resolved]